packages/frameworks-solid-start/src/session.ts

```diff
@@ -0,0 +1,25 @@
+import { AuthHandler, type AuthOptions, type Session } from "@auth/core";
+
+export type GetSessionResult = Promise<Session | null>;
+
+export async function getSession(
```

Comment:
This looks identical to the SvelteKit version:

https://github.com/nextauthjs/next-auth/blob/main/packages/frameworks-sveltekit/src/lib/index.ts#L15

Should probably be in core then, as I can see this coming up again.

A better alternative is to make it so that `AuthHandler` can just take an `action` instead so we don't need a separate `getSession` method.

What do you think @ThangHuuVu?